adapters/handlers/rest/authz/handlers_authz.go

```diff
@@ -638,6 +662,14 @@ func (h *authZHandlers) userExists(user string) bool {
 	return true
 }
 
+// isRootRole validates that enduser do not touch the internal root role
+func isRootRole(name string) error {
+	if name == authorization.Root {
+		return fmt.Errorf("using root role is not allowed")
```

Comment:
I'd say that "using root role" seems to mean "making a request as the 'root' user", whereas sth like `'root' role is read-only` or `modifying 'root' role is not allowed`